contracts/Stake/RewardDelegators.sol

```diff
@@ -352,9 +329,10 @@ contract RewardDelegators is Initializable, Ownable {
         PONDToken = ERC20(_updatedPOND);
     }
 
-    function getClustersWeightedStake(address _cluster) public view returns (uint256) {
-        return clusters[_cluster].weightedStake;
+    function getFullTokenList() public view returns (bytes32[] memory) {
+        return tokenList;
     }
+
     // ************************ IMPORTANT - TO REMOVE ***********************
     // TO remove - for testing only
```

Comment:
remove this comment. Let's let it stay, so that it can be used for debugging purposes later.